hack/ginkgo-e2e.sh

```diff
@@ -74,7 +74,12 @@ else
     prepare-e2e
 
     detect-master >/dev/null
-    KUBE_MASTER_URL="${KUBE_MASTER_URL:-https://${KUBE_MASTER_IP:-}}"
+
```

Comment:
Something like this would be simpler:
```suggestion
    KUBE_MASTER_URL="${KUBE_MASTER_URL:-}"
    if [[ -z "${KUBE_MASTER_URL:-}" && -n "${KUBE_MASTER_IP:-}" ]]
      KUBE_MASTER_URL="https://${KUBE_MASTER_IP}"
    fi
```

Reply:
That's true, thanks!